chef_master/source/chef_license_accept.rst

```diff
@@ -0,0 +1,277 @@
+=====================================================
+Accepting the Chef License
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/chef_license_embedded.rst>`__
+
+This page aims to document how to accept the Chef license for all Chef Software products.
+For an overview of the license, see the `Chef license </chef_license.html>`__ documentation.
+There are two types of license: MLSA and EULA.
+The MLSA applies to customers with a commercial contract with Chef Software, and the EULA covers all other cases.
+
+-----------------------------------------------------
+Accept the Chef MLSA
+-----------------------------------------------------
+There are three ways to accept the Chef MLSA:
+
+#. When running ``chef-<PRODUCT-NAME>-ctl reconfigure`` the Chef MLSA is printed. Type ``yes`` to accept it. Anything other than typing ``yes`` rejects the Chef MLSA, and the upgrade process will exit. Typing ``yes`` adds a ``.license.accepted`` file to the ``/var/opt/<PRODUCT-NAME>/`` directory. As long as this file exists in this directory, the Chef MLSA is accepted and the reconfigure process will not prompt for ``yes``.
+
+#. Run the ``chef-<PRODUCT-NAME>-ctl reconfigure`` command using the ``--accept-license`` option. This automatically types ``yes`` and skips printing the Chef MLSA.
+
+#. Add a ``.license.accepted`` file to the ``/var/opt/<PRODUCT-NAME>/`` directory. The contents of this file do not matter. As long as this file exists in this directory, the Chef MLSA is accepted and the reconfigure process will not prompt for ``yes``.
+
+-----------------------------------------------------
+ Accept the Chef EULA
+-----------------------------------------------------
+
+Workstation Products
```

Comment:
Workstation the product vs. Workstation the classification of products is a bit confusing here. I had to read it over a few times to realize we meant classification

Reply:
@tas50 Will amend this in another PR for this next week.